helpers.go

```diff
@@ -13,3 +13,25 @@ func ApplyJitter(input int) (output int) {
 	deviation := int(0.25 * float64(input))
 	return input - deviation + rand.Intn(2*deviation)
 }
+
+func FindMinAndMax(a []int) (min int, max int) {
+	min = a[0]
+	max = a[0]
+	for _, value := range a {
+		if value < min {
+			min = value
+		}
+		if value > max {
+			max = value
+		}
+	}
+	return min, max
```

Comment:
:thought_balloon: Just wondering as you are using `naked returns`; Is there a rule of thumb of preferring a naked return or when not?